Samples/XCFramework-Validation/XCFramework.xcodeproj/project.pbxproj

```diff
@@ -36,7 +36,7 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
-		1573CAE22BBCA4820018A2B7 /* SentrySwiftUI.xcframework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.xcframework; name = SentrySwiftUI.xcframework; path = Carthage/Build/SentrySwiftUI.xcframework; sourceTree = "<group>"; };
+		1573CAE22BBCA4820018A2B7 /* SentrySwiftUI.xcframework */ = {isa = PBXFileReference; expectedSignature = "AppleDeveloperProgram:97JCY7859U:GetSentry LLC"; lastKnownFileType = wrapper.xcframework; name = SentrySwiftUI.xcframework; path = ../../Carthage/SentrySwiftUI.xcframework; sourceTree = "<group>"; };
```

Comment:
Mmmm, I wonder if having the expected signature may bite us in CI

Reply:
Confirmed no issues here: https://github.com/getsentry/sentry-cocoa/actions/runs/19833508140/job/56825670966